### PR TITLE
CL-1222 Ensure pop-up is not shown if cloud connection is already configured

### PR DESCRIPTION
**Pull request by ianpaschal**
Take 3... action!

### CODE REVIEW

File: plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py [hunk 1]
```diff
@@ -412,10 +412,15 @@ def checkCloudFlowIsPossible(self) -> None:
...
         if active_machine:
             
-            # Check 1: Printer isn't already configured for cloud
+            # Check 1A: Printer isn't already configured for cloud
             if ConnectionType.CloudConnection.value in active_machine.configuredConnectionTypes:
                 Logger.log("d", "Active machine was already configured for cloud.")
                 return
+            
+            # Check 1B: Printer isn't already configured for cloud
+            if active_machine.getMetaDataEntry("cloud_flow_complete", "value") is True:
```
Comment (diegopradogesto) [inline plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py]: Suggestion: Use `parseBool` instead of `is True`

Comment (nallath) [inline plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py]:
```suggestion
            if parseBool(active_machine.getMetaDataEntry("cloud_flow_complete", False)):
```
Importing it should be done by
`from UM.Util import parseBool`

File: plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py [hunk 2]
```diff
@@ -412,10 +412,15 @@ def checkCloudFlowIsPossible(self) -> None:
...
+                Logger.log("d", "Active machine was already configured for cloud.")
+                return
 
             # Check 2: User did not already say "Don't ask me again"
             if active_machine.getMetaDataEntry("show_cloud_message", "value") is False:
```
Comment (nallath) [inline plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py]: Same as above. The default of the metadata entry is False, right (and not "value")

**Comment by ianpaschal** [comment]
I've implemented your feedback, guys. Thank you.

**Comment by ianpaschal** [comment]
I reverted the use of parseBool because it broke everything. I'm not sure why but honestly importing a separate function to parse booleans out of booleans is ugly anyway.

**Pull request merged**